recipe/common.php

```diff
@@ -69,12 +69,19 @@
 
 
 /**
- * Return current release path.
+ * Custom paths.
  */
-set('current_path', function () {
-    $link = run("readlink {{deploy_path}}/current")->toString();
-    return substr($link, 0, 1) === '/' ? $link : get('deploy_path') . '/' . $link;
-});
+set('current_dir', 'current');
+set('release_dir', 'release');
+set('releases_dir', 'releases');
+set('shared_dir', 'shared');
+set('dep_dir', '.dep');
+
+set('current_path', '{{deploy_path}}/{{current_dir}}');
```

Comment:
Now current_path will be returning symlink instead of real path with will lead to bugs. Also does not work with relative symlinks. 

Why did you drop this code?

```
 -set('current_path', function () {
 -    $link = run("readlink {{deploy_path}}/current")->toString();
 -    return substr($link, 0, 1) === '/' ? $link : get('deploy_path') . '/' . $link;
 -});
```